package.json

```diff
@@ -29,7 +29,9 @@
       "hydrogen:run-cell-and-move-down"
     ]
   },
-  "scripts": {},
+  "scripts": {
+    "install": "npm rebuild --build-from-source"
```

Comment:
This will skip the prebuilt goodness and ensure that `zmq-prebuilt` is build from source. This flag is used by `prebuild`.

I may make a PR to `apm` to set this by default.